### PR TITLE
[NFC] Add bazel support to cpu_features

### PR DESCRIPTION
**Pull request by gchatelet**
Partial fix for #221

### CODE REVIEW

**Comment by gchatelet** [comment]
tested with `bazelisk-linux-amd64 test ...`

File: WORKSPACE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```
Comment (Mizux) [inline WORKSPACE:1]: I would add at top:
```
workspace(name = "com_google_cpufeatures")
```

**Review by toor1245 — APPROVED** [review]

**Comment by toor1245** [comment]
Do we need to provide "Build with Bazel" documentation or will it be a separate patch?

**Comment by gchatelet** [comment]
@toor1245 good point ! Let's do it as a second step though, I suspect there will be a few iterations before everything works as correctly. We also need to setup continuous integration.

**Pull request merged**